handout/features/es6_modules.md

```diff
@@ -1,8 +1,33 @@
-# ES6 Modules
+# ECMAScript 2015 Modules
```

Comment:
Eventually the long format is going to be the only way going forward, so we might as well adhere to the standard now. I feel we should just get rid of ES6 altogether and use the long format, early on we can mention ES6 == ECMAScript 2015. Thoughts?